spec/system/provider_interface/provider_views_the_references_tab_on_manage_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+RSpec.feature 'Provider views an application in new cycle' do
+  include CandidateHelper
+  include CycleTimetableHelper
+  include CourseOptionHelpers
+  include DfESignInHelpers
+
+  around do |example|
+    Timecop.freeze(CycleTimetable.apply_opens(2023) + 1.day) do
```

Comment:
Since (very recent) merge of `CycleTimetableHelper` changes you can now call `mid_cycle(2023)` to do this.

Reply:
👍